__tests__/example.test.res.type.ts

```diff
@@ -0,0 +1,9 @@
+'use-strict';
+import * as testlib from '../src/validate';
+
+test('Sync Response fails due to invalid response type', async () => {
+  const exampleSyncRes = require('./example.sync.response.json');
+  const syncErrors = testlib.validate(exampleSyncRes, 'fail');
```

Comment:
You can use `expect(badFn).to.throw()`: https://www.chaijs.com/api/bdd/#method_throw to test for throwing an error.

Reply:
Used the toThrow() function version in jest for this instead